config/phpcs/composer.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "varying-vagrant-vagrants/vvv",
```

I don't think this should be _the_ VVV package, we may want to have a package in the future that lets us setup VVV, at which point we'll have shot ourselves in the foot. We should claim the `vvv` namespace ( and soonish, we're not the first with vvv related packages published)

`vvv/phpcs`?
```suggestion
  "name": "vvv/phpcs",
```

---

That sounds like a good idea